nasl/nasl_scanner_glue.c

```diff
@@ -473,6 +480,11 @@ script_add_preference (lex_ctxt *lexic)
           nasl_perror (lexic, "Preference '%s' already exists\n", name);
           return FAKE_CELL;
         }
+      if (id == nvtpref_id (nvti_pref (script_infos->nvti, i)))
+        {
+          nasl_perror (lexic, "Invalid or already existent preferences id.\n");
```

Comment:
```suggestion
          nasl_perror (lexic, "Invalid or already existent preferences id\n");
```

To match all other nasl_perror not using an ending dot.

Reply:
Dot removed.